MS5611.cpp

```diff
@@ -110,16 +110,16 @@ bool MS5611::isConnected()
 void MS5611::reset()
 {
   command(MS5611_CMD_RESET);
-  delay(3);
+  delayMicroseconds(2800);
```

Comment:
Do you notice the difference in your code?
Are you aware that delay() calls **yield()** internally and afaik delayMicroseconds() does not.
For ESP32 running RTOS this can have some side effects.

Reply:
I ran the code on an altimeter script I wrote for my Arduino Nano and it worked flawlessly. I am not aware of any major differences between delay() and delayMicroseconds() as neither functions disable interrupts.

As for ESP, there [already seems](https://github.com/UncleRus/esp-idf-lib/tree/master/components/ms5611) to be an esp focused library for the MS5611. Maybe this library should try to mainly focus on Arduino devices.